leejihee/first/과제/lyrics.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
여기도 h1 내용으로 바꾸면 좋을 것 같습니다~~
```suggestion
    <title>Lyrics</title>
```